django_polymorphic/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.9.1" %}
+
+package:
+    name: django_polymorphic
+    version: {{ version }}
+
+source:
+    fn: django_polymorphic-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/d/django_polymorphic/django_polymorphic-{{ version }}.tar.gz
+    md5: 1945d107d037d4c9b7fcdd8b14ce0348
+
+build:
+    number: 0
+    preserve_egg_dir: True
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - django
+
+test:
+    imports:
+        - polymorphic
+
+about:
+    home: https://github.com/chrisglass/django_polymorphic
+    license: BSD-3-Clause
```

Comment:
> Ah, right, I forgot. Sorry. Thanks for jogging my memory.

If you can remember everything that is going on in conda-forge you are a robot! :wink: